packages/utils/src/index.ts

```diff
@@ -28,11 +33,11 @@ export type {
   StringOperator,
   FilterOperator,
   TransactionFilter,
-} from './types'
+} from "./types";
 
 export {
   ActionType,
   OrderType,
-} from './types'
+} from "./types";
```

Comment:
Used the VSCode Rome extension to format which explain lots of the formatting changes here. Ended up disabling since I didn't want formatting to make this PR look confusing on other existing files

We should probably discuss formatting later too

https://marketplace.visualstudio.com/items?itemName=rome.rome

Reply:
I have a linear ticket captured to migrate to Biome, rome is deprecated.